scripts/mattermost-postgresql-5.0.sql

```diff
@@ -550,13 +550,13 @@ CREATE TABLE public.teams (
     name character varying(64),
     description character varying(255),
     email character varying(128),
-    type text,
+    type character varying(255),
     companyname character varying(64),
     alloweddomains character varying(500),
     inviteid character varying(32),
     allowopeninvite boolean,
     lastteamiconupdate bigint,
-    schemeid text
+    schemeid varying(26)
```

Comment:
Given that our kind of Id, shouldn't be a problem. In any case I've put it first in the migration so it fails first